readme.md

```diff
@@ -111,6 +111,79 @@ python3 -m nwm_routing -f -V4 test_AnA_V4_HYFeature.yaml
 
 **Note**: The following instructions are for setting up T-Route on a Linux environment (standalone, no MPI). If you are using Windows, please install WSL (Windows Subsystem for Linux) before proceeding.
 
+### T-Route Setup and Testing Guide for Windows Users WITHOUT conda [based on pip and venv - only widely available dependencies]
+
+1. **Install Recommended distro:**
+   - Download and install WSL2 for your Windows OS
+   - We recommend long-term stable (LTS) Ubuntu distribution 22.04 or 20.04 (24.04 not recommended yet)
+   - Open Windows Power Shell and issue
+     > wsl --install Ubuntu-22.04
+   - Enter (root) username and password (2x) of your choice
+     
+2. **Set up venv-based virtual environment:**
+   - From root (the username you created under 1):
+      - Update Linux distro:
+         > sudo apt update
+      - Install pip (package manager):
+         > sudo apt install python3-pip
+      - Install venv:
+         > sudo apt install python3.10-venv
+      - Create a virtual environment for T-route (named here 'troute-env1'):
+         > python3 -m venv troute_env1
+      - Activate your shiny new virtual environment:
+         > source troute_env1/bin/activate
+      - Now, the command prompts in the Shell window should start with (troute-env1)
+ 
+3. **Clone T-route:**
+   - Go to a folder of your choice (here, your home folder) and create a T-route directory
+      > mkdir troute1
+      > cd troute1
+   - Clone a T-route repository (the current main branch is used as an example):
+      > git clone https://github.com/NOAA-OWP/t-route.git
+      > cd troute1
+   - Install python packages per requirements file
+      > pip install -r requirements.txt
+
+4. **Download & build netcdf fortran libraries from UCAR:**
+   - Go to a folder of your choice (here, your home folder) and download the source code:
+      > cd ~
+      > wget https://downloads.unidata.ucar.edu/netcdf-fortran/4.6.1/netcdf-fortran-4.6.1.tar.gz
```

Comment:
Should we specify the version? Is there any way to make it general

Reply:
See above what I wrote on Ubuntu. Why would we not specify the version if we specify many other acceptable versions in the requirements.txt file.

Reply:
Discussed (see Ubuntu long-term stable version issue)